source/Module/Rule.nxFileLine/Convert/Data.ps1

```diff
@@ -28,32 +32,37 @@ data doesNotContainPattern
         'active = yes'                                              = '\s*active\s*=\s*no|active=yes|#\s*active\s*=.*'
         'Unattended-Upgrade::Remove-Unused-Dependencies "true";'    = '\s*Unattended-Upgrade::Remove-Unused-Dependencies\s*("false"|false|true).*|#\s*Unattended-Upgrade::Remove-Unused-Dependencies.*'
         'Unattended-Upgrade::Remove-Unused-Kernel-Packages "true";' = '\s*Unattended-Upgrade::Remove-Unused-Kernel-Packages\s*("false"|false|true).*|#\s*Unattended-Upgrade::Remove-Unused-Kernel-Packages.*'
-        'session required pam_lastlog.so showfailed'                = '\s*session\s*(?!required)\w*\s*pam_lastlog\.so.*|#\s*session\s*\w*\s*pam_lastlog\.so.*'
+        'session required pam_lastlog.so showfailed'                = '^\s*session\s*(?!required)\w*\s*pam_lastlog\.so.*|#\s*session\s*\w*\s*pam_lastlog\.so.*|^\s*session(?:\t+|\s{2,})required(?:\t+|\s{2,})pam_lastlog\.so.*'
         'ucredit=-1'                                                = '^#\s*ucredit.*$|^ucredit\s*=\s*(?!-1\b)\w*$'
+        'ucredit = -1'                                              = '^#\s*ucredit.*$|^ucredit\s*=\s*(?!-1\b)\w*$'
```

Comment:
can we either line up the "=" or not throughout this block for readability please